core/src/main/scala/sttp/openai/OpenAI.scala

```diff
@@ -1082,27 +1084,27 @@ class OpenAI(authToken: String) {
 
 }
 
-private object OpenAIUris {
-  private val ImageBase: Uri = uri"https://api.openai.com/v1/images"
-  private val AudioBase: Uri = uri"https://api.openai.com/v1/audio/"
+private class OpenAIUris(val BaseUri: Uri) {
```

Comment:
[minor] shouldn't this be lower-case? `baseUri`

Reply:
could be, in this case also `imageBase` and `audioBase`, right?

Reply:
`ImageBase` in the old code was a constant, this is a variable, so it's written using lowercase I guess? :)